front-end/src/components/RightToolBar/FriendsList.js

```diff
@@ -2,7 +2,15 @@ import React, { useState } from 'react';
 import { Grid, Container, Paper, IconButton } from '@material-ui/core';
 import IndividualFriend from './IndividualFriend.js';
 
-const FriendsList = ({ friendsList, friends }) => {
+const FriendsList = ({ friends, userInput }) => {
```

Comment:
is userInput used?

Reply:
Hey Monica. not currently, but i'm in the process of setting up the use of it now.